quinn-proto/src/connection/paths.rs

```diff
@@ -45,8 +58,17 @@ impl PathData {
             congestion,
             challenge: None,
             challenge_pending: false,
+            validated: false,
+            total_sent: 0,
+            total_recvd: 0,
         }
     }
+
+    /// Indicates whether we're a server that hasn't validated the peer's address and hasn't
+    /// received enough data from the peer to permit sending `bytes_to_send` additional bytes
+    pub fn anti_amplification_blocked(&self, bytes_to_send: u16) -> bool {
```

Comment:
Really worthwhile to stick with `u16` here? Guess it's just moving the cast around.

Reply:
Given that we'll in the very near future be passing multiple MTUs, I suppose widening this is sensible.